src/main/java/frc/robot/subsystems/conditioners/ReverseConditioner.java

```diff
@@ -1,7 +1,7 @@
 package frc.robot.subsystems.conditioners;
 
 public class ReverseConditioner implements DriveConditioner {
-  private boolean reversed = false;
+  private boolean reversed = true;
```

Comment:
Why does reversed default to true?  It should default to false and the backwards robot should be fixed elsewhere.